spec/helpers/tweets_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe TweetsHelper, :vcr do
+
+  describe 'Favorite button' do
+
+    it 'returns the html for the button' do
+      tweet = TwitterClient.new.search('463440424141459456').first
+
+      expect(favorite_button(tweet)).to have_css(:span)
```

Comment:
Para este caso no me parece mal poner dos expects en el mismo `it`. Más allá de que normalmente se intenta no hacerlo